src/Card/CardImageCap.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import CardContext from './CardContext';
+
+const CardImageCap = React.forwardRef(({
+  src,
+  logoSrc,
+  className,
+}, ref) => {
+  const { orientation } = useContext(CardContext);
+  const wrapperClassName = `pgn__card-wrapper-image-cap ${orientation}`;
+
+  return (
+    <div className={classNames(className, wrapperClassName)} ref={ref}>
+      <img className="pgn__card-image-cap" src={src} alt={`card-cap-${orientation}`} />
```

Comment:
Does `card-cap-${orientation}` make sense as the `alt` text for an image? It's not really describing what the image is. Should we update this component to allow users to pass in an optional, custom `alt` for the image?

Similar question for the logo image below.

Reply:
Agree, the alt texts are not very good here, and it's also a good idea to add additional props. Updated it